samples/DrawWithImageSharp/Program.cs

```diff
@@ -56,6 +56,12 @@ public static void Main(string[] args)
             FontFamily arabicFont = SystemFonts.Get("Dubai");
 
             FontFamily tahoma = SystemFonts.Get("Tahoma");
+
```

Comment:
Ignore this stuff. I plan on creating a clean sample project to demonstrate multiple features.